mppm/virtual-players/virtual-players-layout.md

```diff
@@ -0,0 +1,35 @@
+---
+id: virtual-players-layout
+title: Change the layout of a Virtual Player
+description: Learn how to change the layout of a Vrtual Player window.
+---
+
+You can display and use the following parts of the [Unity Interface](https://docs.unity3d.com/Manual/UsingTheEditor.html) in a Virtual Player:
```

Comment:
Do you need a blank line between the paragraph and the list for it to render correctly?

Reply:
How about something like "The Virtual Player can display the following parts of the Unity Interface"? That you can use them once they're displayed is, I think, obvious.